definitions/ext-rclusterhost/summary_metrics.yml

```diff
@@ -0,0 +1,24 @@
+rclusterCpuUsage:
+  title: CPU usage (%)
+  unit: PERCENTAGE
+  query:
+    select: average(rcluster.container.cpu.percentused)*100
+    from: Metric
+    facet: rfc460Hostname
+    eventId: entity.guid
+rclusterMemoryUsage:
+  title: Memory usage (%)
+  unit: PERCENTAGE
+  query:
+    select: average(rcluster.container.memory.percentused)*100
+    from: Metric
+    facet: rfc460Hostname
+    eventId: entity.guid
+rclusterStorageUsage:
+  title: Storage usage (%)
+  unit: PERCENTAGE
+  query:
+    select: average(rcluster.container.executor.disk.percentused)*100
+    from: Metric
+    facet: rfc460Hostname
```

Comment:
As far as i know there's no facet in summary metrics ([docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#nrdb-query)). It will use the `eventId` to facet by.

Reply:
updated